loopj/jquery-tokeninput
# Permit both theme and custom classes

#### What's this PR do?

Permits both themes and custom classes.  Prior to this pull request, it was not possible to have a theme take effect if you were also trying to pass in any custom classes, e.g.:

``` javascript
$('.token-autocomplete').tokenInput({
  theme: 'bootstrap',
  classes: {
    tokenList: 'token-input-list-bootstrap form-control'
  }
});
```
#### How should this be manually tested?

Try all four possibilities to ensure they work as expected.
1. No `theme` or `classes` option passed in.
2. Just a `theme` option passed in.
3. Just a `classes` option passed in.
4. Both a `theme` and `classes` option passed in.
#### Any background context you want to provide?

I'm trying to use a theme, but I'd also like to attach an unrelated bootstrap class to the rendered token input.
#### Screenshots

![jquery-tokeninput](https://cloud.githubusercontent.com/assets/169093/5988541/eec7bc72-a921-11e4-8691-ec5070e2795b.png)
#### What gif best describes this PR or how it makes you feel?

![high five!](http://i.imgur.com/YqcbU59.gif)
